src/client/package.json

```diff
@@ -5,13 +5,8 @@
   "dependencies": {
     "@loadable/component": "^5.12.0",
     "@testing-library/react": "^10.0.4",
-    "@testing-library/react-hooks": "^3.2.1",
```

Comment:
Not taking any of those off as it seems they are needed dependencies by react/tests/enzyme

Reply:
missing glob and rimraf for translations script